src/components/Header.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div class="container">
+    <h1 class="title">To do list</h1>
+    <button @click="addTodo" class="Add"><img :src="Addplus" /></button>
+  </div>
+</template>
+
+<script setup lang="ts">
+import Addplus from "../assets/addplus.png";
+const emit = defineEmits<{
+  (e: "todoAdded", value: string): void;
+}>();
+
+function addTodo() {
+  emit("todoAdded", "Add-one-todo");
```

Comment:
Why do you need to emit "Add-one-todo" string can you just emit the todoAdded action without any parameters?